gitlab-pipeline/stage/test.yml

```diff
@@ -181,9 +186,9 @@ test:backend-integration:azblob:enterprise:
   before_script:
     # Default value, will later be overwritten if successful
     - echo "failure" > /JOB_RESULT.txt
+
+    # Make sure docker interacts with dockerd via socket
     - unset DOCKER_HOST
-    - unset DOCKER_TLS_VERIFY
-    - unset DOCKER_CERT_PATH
```

Comment:
Nice clean-up.

We also need to remove them from [here](https://github.com/mendersoftware/mender-qa/blob/3e2913a6f1fce4823d46b24443151e6b7e6d163c/gitlab-pipeline/internal-variables.yml#L13-L14) and [here](https://github.com/mendersoftware/mender-qa/blob/3e2913a6f1fce4823d46b24443151e6b7e6d163c/gitlab-pipeline/stage/test.yml#L296-L297).

Reply:
I don't see `gitlab-pipeline/internal-variables.yml` is included in any of our templates. All [those](https://github.com/mendersoftware/mender-qa/blob/3e2913a6f1fce4823d46b24443151e6b7e6d163c/gitlab-pipeline/internal-variables.yml#L8-L15) variables are for accessing `dockerd` via tcp, which we don't use, and probably all of them can be deleted then.

Just for the reference: `unset DOCKER_HOST` is required, because the variable is set by GitLab

Reply:
Due to `internal-variables.yml` is not included by the template I would suggest to deal with it separately.

Reply:
It is included [here](https://github.com/mendersoftware/mender-qa/blob/0ef9e9c64fed41b1eee20287fab4b84072bb2687/.gitlab-ci.yml#L100).

Reply:
Follow-up at:
* https://github.com/mendersoftware/mender-qa/pull/652